tools/database/wowhead_tooltips.go

```diff
@@ -41,7 +41,7 @@ func NewWowheadSpellTooltipManager(filePath string) *WowheadTooltipManager {
 	}
 }
 
-type Stats [35]float64
+type Stats [41]float64
```

Comment:
You added one more stat but here you add 6 new indices. Where is this coming from?

Reply:
You're right, might have added too many. There was another Stats float that did need to be 41. Will look into it.

Reply:
I don't know how this was 35 before, but we must not have generated tooltips since 4-5 new stats were added because `proto.Stat_StatMastery` has index 40 and `proto.Stat_StatShadowResistance` has 33.

Other stats from proto:
```
	Stat_StatShadowResistance  Stat = 33
	Stat_StatBonusArmor        Stat = 34
	Stat_StatRunicPower        Stat = 35 // Placeholder: not used yet.
	Stat_StatBloodRune         Stat = 36 // Placeholder: not used yet.
	Stat_StatFrostRune         Stat = 37 // Placeholder: not used yet.
	Stat_StatUnholyRune        Stat = 38 // Placeholder: not used yet.
	Stat_StatDeathRune         Stat = 39 // Placeholder: not used yet.
	Stat_StatMastery           Stat = 40
```

Reply:
And the tools lenght was 35 because we have 35 stats (last index is 34 with BonusArmor)

Reply:
Fixed in [09816d6](https://github.com/wowsims/cata/pull/1/commits/09816d62bb9524e573d1396cf4446f6d3910be00)